wordpress-mobile/WordPress-Android
# Add adaptive icon

This attempts to address #5529. It adds app icon back and foregrounds, and an adaptive asset.

The ongoing conversation right now is whether this looks good in the shapes other than the round version. This particular version of the icon uses a 3dp padding around the WordPress logo. Should we try a 4dp version?

Fixes #5529 

To test:

- Build this branch on a Oreo device
- Test that the icon looks good
- Try and "jiggle" the icon to see the shape inside parallax
- If you have the Google Pixel launcher, you can long press the homescreen to test other icon shapes, like square and squircle

Screenshots:

![2017-11-15 10_30_25](https://user-images.githubusercontent.com/1204802/32828699-30c5b54c-c9f0-11e7-8614-a147b797d88f.gif)

![screenshot_20171115-102552](https://user-images.githubusercontent.com/1204802/32828714-36775838-c9f0-11e7-808e-80a32b877434.png)

![screenshot_20171115-102535](https://user-images.githubusercontent.com/1204802/32828718-3844358c-c9f0-11e7-8785-278065b1f6e3.png)

![screenshot_20171115-102525](https://user-images.githubusercontent.com/1204802/32828721-3a2f1678-c9f0-11e7-8d64-6847ca5009f5.png)
![screenshot_20171115-102501](https://user-images.githubusercontent.com/1204802/32828723-3bb35324-c9f0-11e7-96ef-4c658a107ad0.png)

![screenshot_20171115-102512](https://user-images.githubusercontent.com/1204802/32828729-3dd38c00-c9f0-11e7-8e65-3847bbbd1079.png)
